src/applications/disability-benefits/all-claims/content/secondaryOtherSources.jsx

```diff
@@ -0,0 +1,46 @@
+import React from 'react';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+export const otherSourcesDescription = (
+  <div>
+    <h3>Other sources of information</h3>
+    <p>
+      If you were treated at a military or private facility for this event, or
+      reported the event to the authorities or other parties, we can help you
+      gather supporting information from them for your claim.
+    </p>
+    <p>
+      If you have supporting (lay) statements from friends, family, or clergy,
+      or have copies of reports from authorities, you‘ll be able to upload those
+      later in the application.
+    </p>
+    <p>Would you like us to help you gather this information?</p>
```

Comment:
Could you put the contents of this `<p>` as the `ui:title` for the field so you don't have to have `'ui:title': ' '`?

Reply:
Done!